scalding-core/src/main/scala/com/twitter/scalding/Config.scala

```diff
@@ -51,6 +51,20 @@ trait Config extends Serializable {
     }
 
   /**
+   * Add files to be localized to the config. Intended to be used by user code.
+   * @param cachedFiles CachedFiles to be added
+   * @return new Config with cached files
+   */
+  def addDistributedCacheFiles(cachedFiles: CachedFile*): Config =
```

Comment:
yes, I want to have some API to work with `CachedFile` for backward capability, because I sure that many people already use `DistributedCacheFile` (`DistributedCacheFile` will work fine with execution because `Execution` has method `getMode/getConfigAndMode`). 

```
def addDistributedCacheFiles(uris: URII): Config
def addDistributedCacheFiles(files: File*): Config
```

make sense to have it.